### DISCUSSION TITLE
QAnything本地部署，是否有注册/登录功能，以及是否有管理员后台对话管理功能？

### DISCUSSION THREAD

**Discussion by brwang** [netease-youdao/QAnything#486] [Q&A]
QAnything本地部署，是否有注册/登录功能，以及是否有管理员后台对话管理功能？

**Comment by TatyOko28** [comment]
QAnything本地部署是否支持注册/登录功能和管理员后台对话管理功能，取决于其版本以及项目实现的具体细节。如果这些功能未内置，你可以通过扩展代码或结合第三方工具来实现。
